src/robusta/runner/telemetry.py

```diff
@@ -1,6 +1,19 @@
+from collections import defaultdict
 from pydantic.main import BaseModel, Optional
-from datetime import datetime
+from typing import Dict
+
+class SinkInfo(BaseModel):
+    type: str = 'None'
+    findings_count: int = 0
 
 class Telemetry(BaseModel):
+    account_id: str = 'None'
```

Comment:
Why do we put that default value here?
I think "None" is a bad default, because it's confusing

Reply:
good idea. thanks